keyboards/ergodash/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+ifndef QUANTUM_DIR
+	include ../../../../Makefile
```

Comment:
This block is a holdover from the old make system. If you could remove it (an empty file here is fine)